tests/bundle/Command/ExportCommandTest.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Tests\Bundle\Personalization\Command;
+
+use EzSystems\EzRecommendationClient\Exception\InvalidArgumentException;
+use EzSystems\EzRecommendationClient\Exception\MissingExportParameterException;
+
+final class ExportCommandTest extends AbstractCommandTestCase
+{
+    protected function setUp(): void
+    {
+        parent::setUp();
+    }
+
+    protected static function getCommandName(): string
+    {
+        return 'ibexa:recommendation:run-export';
+    }
+
+    public function testThrowExceptionWhenInvalidSiteAccessIsPassed(): void
+    {
+        $siteAccess = 'undefined_siteaccess';
+        $this->expectException(InvalidArgumentException::class);
+        $this->expectExceptionMessage(
+            sprintf('SiteAccess %s doesn\'t exists', $siteAccess)
+        );
+        $this->commandTester->execute(
+            [
+                '--siteaccess' => $siteAccess,
+            ]
+        );
+    }
+
+    /**
+     * @param array<array<array<string>|string>> $parameters
+     *
+     * @dataProvider provideForTestCommandThrowExceptionWhenRequiredParametersAreMissing
+     */
+    public function testThrowExceptionWhenRequiredParametersAreMissing(
+        array $parameters,
+        string $expectExceptionMessage
+    ): void {
+        $this->expectException(MissingExportParameterException::class);
+        $this->expectExceptionMessage($expectExceptionMessage);
+        $this->commandTester->execute($parameters);
+    }
+
+    /**
+     * @return array<array<array<string>|string>>.
```

Comment:
More specifically, in the end those are two-element arrays, with an array as first and a string as second element.
```suggestion
     * @return array<array{array<string>, string}>.
```